NuRadioReco/modules/efieldToVoltageConverter.py

```diff
@@ -222,10 +224,11 @@ def run(self, evt, station, det):
                 plt.show()
             if trace_object is None:  # this happens if don't have any efield for this channel
                 # set the trace to zeros
-                channel.set_trace(np.zeros(trace_length_samples), 1. / time_resolution)
```

Comment:
I think this change and the two changes below are not required. This should put it back into the expected functionality. 
The bug was that the first channel that was considered didn't have any efield. But I think we can exclude that we have an event with no efields at all. So at least one efield is present such that  `trace_lenght_samples` and `time_resolution` and `times_min` are set